docs/user_guide/bank/community_product_drives.md

```diff
@@ -1 +1,25 @@
-Not yet written
\ No newline at end of file
+# Product Drive Participants
+
+You may have organized a product drive in your community and would like to manage the participants.
+
+To manage product drive participants, click on " Product Drive Participants" under the "Community" section.
+
+![Product Drive Participants](images/community/product_drive/product_drive_page.jpg)
+
+If you have done product drives before, the page will be populated with the participants' details. 
+
+To add a new participant, click on the "+ New Product Drive Participant" button, add their details including the business name, contact name, phone, email, address and comments.
+
```

Comment:
Add:  "You can also add new participants "on the fly" as you enter the donations,  through the new donation screen 


(just provide a link to the essentials_donations for now,  but please add a "todo" to link to the new donation section within it -- donations isn't written yet.)